assets/test/defaultPackage/Tests/PackageTests/PackageTests.swift

```diff
@@ -112,11 +112,12 @@ struct MixedSwiftTestingSuite {
 }
 #endif
 
-#if swift(>=6.1)
-@Test func testAttachment() throws {
-  Attachment("Hello, world!", named: "hello.txt").attach()
-}
-#endif
+// Disabled until Attachments are formalized and released.
+// #if swift(>=6.1)
+// @Test func testAttachment() throws {
+//   Attachment("Hello, world!", named: "hello.txt").attach()
```

Comment:
I can't comment on the line itself above, but I'd also suggest you remove `@_spi(Experimental)` from the `import Testing` above, to help ensure you don't accidentally rely on SPI in the future.